myteamUi.py

```diff
@@ -0,0 +1,33 @@
+# not currently used
+from tkinter import *
+
+
+class MyTeamUi(Frame):
+    def __init__(self, master=None):
+		"""
+			inits a new MyTeamUi object to suggest CharacterInstances for the user's team
+		"""
+        Frame.__init__(self, master)
+        self.createwidgets()
+
+    def compare_char_f(self):
+        pass
+
+    def suggest_team_f(self):
+        pass
+
+    def createwidgets(self):
```

Comment:
create_widgets
"""
create_widgets creates display elements for myteam scren
 """